File: publicq-main/src/PublicQ.API/Models/Requests/ResetPasswordRequest.cs
namespace PublicQ.API.Models.Requests;

/// <summary>
/// Reset password request model.
/// </summary>
public class ResetPasswordRequest
{
    /// <summary>
    /// User's email address.
    /// </summary>
    public required string Email { get; set; }
    
    /// <summary>
    /// User's password reset token.
    /// </summary>
    public required string Token { get; set; }
    
    /// <summary>
    /// User's new password.
    /// </summary>
    public required string NewPassword { get; set; }
}

File: publicq-main/client/src/models/reset-password-request.ts
/**
 * Reset password request model.
 */
export interface ResetPasswordRequest {
  /**
   * User's email address.
   */
  email: string;
  
  /**
   * User's password reset token.
   */
  token: string;
  
  /**
   * User's new password.
   */
  newPassword: string;
}
